wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/TaskSectionElement.scala

```diff
@@ -1,4 +1,4 @@
 package wdl.model.draft3.elements
 
 /** Supertype for elements contained in a task body to simplify task conversions. */
-trait TaskBodyElement extends LanguageElement
+trait TaskSectionElement extends LanguageElement
```

Comment:
ToL since the inheriting case classes are all pretty small, it might be a good idea at some point to move them all into the `TaskSectionElement` file and make this a `sealed trait`

Reply:
I think I'll wait until I've picked up your changes to inputs and outputs.

Reply:
Good point! It is possible, but I guess if we want to do the same with workflow too we'd need to combine them all into the same file, which doesn't seem great. I'd leave it as it is for now!